src/server/services/refreshAllReadingViews.js

```diff
@@ -9,12 +9,25 @@ const { log } = require('../log');
 const refreshDailyReadingViews = require('./refreshReadingViews').refreshReadingViews;
 const { refreshHourlyReadingViews } = require('./refreshHourlyReadingViews');
 
+/** 
+* @deprecated Please use the new refreshAllReadingViews() function
+		      where hourly readings are refreshed before daily readings
+**/
+async function refreshAllReadingViewsOld() {
+
+	log.info('Refreshing All Reading Views...');
+	await Promise.all([refreshDailyReadingViews(), refreshHourlyReadingViews()]);
+	log.info('...Views Refreshed!');
+}
 
 async function refreshAllReadingViews() {
```

Comment:
The tests are complaining that this is not a function. This needs to be resolved. After that we can see if any are still failing.